src/iact_estimator/core.py

```diff
@@ -167,9 +169,13 @@ def check_input_configuration(config):
             " region (n_off_regions) should be used."
         )
         is_valid = False
-    if config["sum_trigger"] and (config["zenith_performance"] == "mid"):
-        logger.warning("This functionality has not been yet implemented.")
-        # raise NotImplementedError("This functionality has not been yet implemented.")
+    if config["sum_trigger"] and (config["zenith_range"] == "mid"):
+        is_valid = False
+        raise NotImplementedError(
+            "MAGIC SUM trigger at mid zenith range has not been yet implemented."
+        )
+    if ("LST" in performance_metadata) and config["sum_trigger"]:
```

Comment:
`performance_metadata` is not assigned when not using `performance_data`, so I got an error when I launched the script without custom dataset

Reply:
correct, this is related to the thread above